### ISSUE TITLE
Support for melative, AniList, and Hummingbird.

### ISSUE THREAD

**Issue by CatmanIX** [japanesemediamanager/jmmserver#156]
Probably be awhile for any of this to get integrated, but having options for connecting to [AniList](http://anilist.co/), [melative](http://melative.com/), and [Hummingbird](http://hummingbird.me/) would be nice for those who use such services.

Hummingbird has already been requested (#63), but i figured i'd mention it again while i'm mentioning the others since the topic hasn't been touched in awhile.

**Comment by jmediamanager** [comment]
I am finishing up work on Trakt over the next week or so.
Hummingbird would probably be next (although not for quite a while) as it looks to be the most popular of the ones you mentioned

**Comment by CatmanIX** [comment]
Yeah, i figured it would be awhile before this got done and that you were probably already looking into it, figured I'd just make a ticket to open discussion.

Hummingbird definitely seems to be more popular atm, but AniList has been gaining a lot more traction lately. I've been seeing many more people talking about AniList lately than i do Hummingbird. Possibly might have something to do with it's connection to AniChart.

Though, if you're still thinking about redoing the service linking system with something based on XEM (#48), it might be prudent to wait and do it all with that (it should be easier to add new services with it). I was actually going to take a look at it in the next couple days and see how easy it would be to adapt to JMMs needs.

**Comment by ElementalCrisis** [comment]
AniList has been requested before, #64 and so has HummingBird, #63. Will keep this issue open for Melative.
